### DISCUSSION TITLE
How to make horizontal scroll on mobile devices with Smooth-scrollbar

### DISCUSSION THREAD

**Discussion by kaliostro1979** [idiotWu/smooth-scrollbar#338] [Q&A] [answered]
Hi. I have some web page with horizontal scroll. In PC version when i scroll my page scrolling horizontal and it's great!! But i want to make horizontal scroll touch devices but not from down to up, just from left to right. Please help with this. Thanks a lot for help. You can check it in my site http://new-site.frigatestudio.com/

**Comment by dolphin-wood** [comment, marked as answer]
See https://github.com/idiotWu/smooth-scrollbar/issues/181#issuecomment-458581737

**Comment by kaliostro1979** [comment]
Thanks for response. As far as I understand, I need to use an additional plugin, in this case "invertDelta"?

**Comment by kaliostro1979** [comment]
It's just that I'm new in JS and maybe I don't understand something, if you can explain in more detail, please, how can I do it with "Smooth-scroll" plugin. Thank you

**Comment by dolphin-wood** [comment]
You can refer to the demo at https://codepen.io/idiotWu/pen/KJNYye?editors=0010
